packages/dropdowns/src/styled/items/StyledItemMeta.ts

```diff
@@ -6,19 +6,26 @@
  */
 
 import styled from 'styled-components';
-import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
-import MenuStyles from '@zendeskgarden/css-menus';
+import { retrieveComponentStyles, DEFAULT_THEME, getColor } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'dropdowns.item_meta';
 
+interface IStyledItemMetaProps {
+  isCompact?: boolean;
+}
+
 /**
  * Accepts all `<span>` props
  */
 export const StyledItemMeta = styled.span.attrs({
   'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION,
-  className: MenuStyles['c-menu__item__meta']
-})`
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledItemMetaProps>`
+  display: ${props => (props.isCompact ? 'none' : 'block')};
+  line-height: ${props => props.theme.lineHeights.sm};
```

Comment:
Same as https://github.com/zendeskgarden/react-components/pull/600#discussion_r371514754

Since this is meant to apply to text, why wouldn't we use these themed values?

Reply:
Moved from our offline conversation.